M2/Macaulay2/packages/Macaulay2Doc/functions/select-doc.m2

```diff
@@ -274,10 +276,10 @@ doc ///
     selectPairs(n, x, f)
   Inputs
     n:ZZ
-    x:HashTable -- must be immutable
+    x:{HashTable, BasicList}
     f:Function
   Outputs
-    :HashTable
+    :{HashTable, Sequence}
```

Comment:
Why a Sequence?

Reply:
Oh I guess this should change after your latest commit.

Reply:
Yeah -- this was based on the current "always a sequence" behavior.  The current draft always returns a `List` object, but I like the idea of behaving like `apply` and returning an instance of the same class.